{
  "Title": "Security update for python-numpy",
  "Tracking": {
    "ID": "SUSE-SU-2019:2462-1",
    "Status": "Final",
    "Version": "1",
    "InitialReleaseDate": "2019-09-25T14:43:07Z",
    "CurrentReleaseDate": "2019-09-25T14:43:07Z",
    "RevisionHistory": [
      {
        "Number": "1",
        "Date": "2019-09-25T14:43:07Z",
        "Description": "current"
      }
    ]
  },
  "Notes": [
    {
      "Text": "Security update for python-numpy",
      "Title": "Topic",
      "Type": "Summary"
    },
    {
      "Text": "This update for python-numpy fixes the following issues:\n\nNon-security issues fixed:\n\n- Updated to upstream version 1.16.1. (bsc#1149203) (jsc#SLE-8532)\n",
      "Title": "Details",
      "Type": "General"
    },
    {
      "Text": "The CVRF data is provided by SUSE under the Creative Commons License 4.0 with Attribution for Non-Commercial usage (CC-BY-NC-4.0).",
      "Title": "Terms of Use",
      "Type": "Legal Disclaimer"
    }
  ],
  "ProductTree": {
    "Relationships": [
      {
        "ProductReference": "python-numpy-1.16.1-4.8.1",
        "RelatesToProductReference": "SUSE Linux Enterprise Module for Basesystem 15",
        "RelationType": "Default Component Of"
      },
      {
        "ProductReference": "python2-numpy-1.16.1-4.8.1",
        "RelatesToProductReference": "SUSE Linux Enterprise Module for Basesystem 15",
        "RelationType": "Default Component Of"
      },
      {
        "ProductReference": "python2-numpy-devel-1.16.1-4.8.1",
        "RelatesToProductReference": "SUSE Linux Enterprise Module for Basesystem 15",
        "RelationType": "Default Component Of"
      },
      {
        "ProductReference": "python3-numpy-1.16.1-4.8.1",
        "RelatesToProductReference": "SUSE Linux Enterprise Module for Basesystem 15",
        "RelationType": "Default Component Of"
      },
      {
        "ProductReference": "python3-numpy-devel-1.16.1-4.8.1",
        "RelatesToProductReference": "SUSE Linux Enterprise Module for Basesystem 15",
        "RelationType": "Default Component Of"
      },
      {
        "ProductReference": "python-numpy_1_16_1-gnu-hpc-1.16.1-4.8.1",
        "RelatesToProductReference": "SUSE Linux Enterprise Module for High Performance Computing 15",
        "RelationType": "Default Component Of"
      },
      {
        "ProductReference": "python2-numpy-gnu-hpc-1.16.1-4.8.1",
        "RelatesToProductReference": "SUSE Linux Enterprise Module for High Performance Computing 15",
        "RelationType": "Default Component Of"
      },
      {
        "ProductReference": "python2-numpy-gnu-hpc-devel-1.16.1-4.8.1",
        "RelatesToProductReference": "SUSE Linux Enterprise Module for High Performance Computing 15",
        "RelationType": "Default Component Of"
      },
      {
        "ProductReference": "python3-numpy-gnu-hpc-1.16.1-4.8.1",
        "RelatesToProductReference": "SUSE Linux Enterprise Module for High Performance Computing 15",
        "RelationType": "Default Component Of"
      },
      {
        "ProductReference": "python3-numpy-gnu-hpc-devel-1.16.1-4.8.1",
        "RelatesToProductReference": "SUSE Linux Enterprise Module for High Performance Computing 15",
        "RelationType": "Default Component Of"
      }
    ]
  },
  "References": [
    {
      "URL": "https://www.suse.com/support/update/announcement/2019/suse-su-20192462-1/",
      "Description": "Link for SUSE-SU-2019:2462-1"
    },
    {
      "URL": "https://lists.suse.com/pipermail/sle-security-updates/2019-September/005966.html",
      "Description": "E-Mail link for SUSE-SU-2019:2462-1"
    },
    {
      "URL": "https://www.suse.com/support/security/rating/",
      "Description": "SUSE Security Ratings"
    }
  ],
  "Vulnerabilities": [
    {
      "CVE": "CVE-2019-6446",
      "Description": "** DISPUTED **   An issue was discovered in NumPy 1.16.0 and earlier. It uses the pickle Python module unsafely, which allows remote attackers to execute arbitrary code via a crafted serialized object, as demonstrated by a numpy.load call. NOTE: third parties dispute this issue because it is  a behavior that might have legitimate applications in (for example)  loading serialized Python object arrays from trusted and authenticated  sources.",
      "Threats": [
        {
          "Type": "Impact",
          "Severity": "important"
        }
      ],
      "References": [
        {
          "URL": "https://www.suse.com/security/cve/CVE-2019-6446.html",
          "Description": "CVE-2019-6446"
        },
        {
          "URL": "https://bugzilla.suse.com/1122208",
          "Description": "SUSE Bug 1122208"
        }
      ],
      "ProductStatuses": [
        {
          "Type": "Fixed",
          "ProductID": [
            "SUSE Linux Enterprise Module for Basesystem 15:python-numpy-1.16.1-4.8.1",
            "SUSE Linux Enterprise Module for Basesystem 15:python2-numpy-1.16.1-4.8.1",
            "SUSE Linux Enterprise Module for Basesystem 15:python2-numpy-devel-1.16.1-4.8.1",
            "SUSE Linux Enterprise Module for Basesystem 15:python3-numpy-1.16.1-4.8.1",
            "SUSE Linux Enterprise Module for Basesystem 15:python3-numpy-devel-1.16.1-4.8.1",
            "SUSE Linux Enterprise Module for High Performance Computing 15:python-numpy_1_16_1-gnu-hpc-1.16.1-4.8.1",
            "SUSE Linux Enterprise Module for High Performance Computing 15:python2-numpy-gnu-hpc-1.16.1-4.8.1",
            "SUSE Linux Enterprise Module for High Performance Computing 15:python2-numpy-gnu-hpc-devel-1.16.1-4.8.1",
            "SUSE Linux Enterprise Module for High Performance Computing 15:python3-numpy-gnu-hpc-1.16.1-4.8.1",
            "SUSE Linux Enterprise Module for High Performance Computing 15:python3-numpy-gnu-hpc-devel-1.16.1-4.8.1"
          ]
        }
      ],
      "CVSSScoreSets": {}
    }
  ]
}